python/lsst/cp/pipe/makeBrighterFatterKernel.py

```diff
@@ -336,14 +336,14 @@ def averageCorrelations(self, xCorrList, name):
 
         Parameters
         ----------
-        xCorrList : `list` [`numpy.array`]
+        xCorrList : `list` [`numpy.array`, (N, N)]
```

Comment:
Hmm... this is so non-standard I'm not sure which is better. 😕 Good question for `#dm-docs-support`?

Reply:
Moved to text descriptions of dimensionality for these cases.